src/core/ext/filters/http/message_compress/message_compress_filter.c

```diff
@@ -171,17 +200,14 @@ static grpc_error *process_send_initial_metadata(
   error = grpc_metadata_batch_add_tail(
       exec_ctx, initial_metadata, &calld->accept_encoding_storage,
       GRPC_MDELEM_ACCEPT_ENCODING_FOR_ALGORITHMS(
-          channeld->supported_compression_algorithms &
-          GRPC_MESSAGE_COMPRESSION_ALGORITHM_MASK));
```

Comment:
Because the bits for stream compression algorithms is no longer being captured by `channeld->supported_compression_algorithms`. They are now in `channeld->supported_stream_compression_algorithms`. So right now we don't need mask any of them.